{
  "affected": [
    {
      "database_specific": {
        "source": "https://github.com/bitnami/vulndb/tree/main/data/gitlab/BIT-gitlab-2021-22200.json"
      },
      "package": {
        "ecosystem": "Bitnami",
        "name": "gitlab",
        "purl": "pkg:bitnami/gitlab"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "12.6.0"
            },
            {
              "fixed": "13.8.7"
            },
            {
              "introduced": "13.9.0"
            },
            {
              "fixed": "13.9.5"
            },
            {
              "introduced": "13.10.0"
            },
            {
              "fixed": "13.10.1"
            }
          ],
          "type": "SEMVER"
        }
      ],
      "severity": [
        {
          "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
          "type": "CVSS_V3"
        }
      ]
    }
  ],
  "aliases": [
    "CVE-2021-22200"
  ],
  "database_specific": {
    "cpes": [
      "cpe:2.3:a:gitlab:gitlab:*:*:*:*:community:*:*:*",
      "cpe:2.3:a:gitlab:gitlab:*:*:*:*:enterprise:*:*:*",
      "cpe:2.3:a:gitlab:gitlab:13.10.0:*:*:*:community:*:*:*",
      "cpe:2.3:a:gitlab:gitlab:13.10.0:*:*:*:enterprise:*:*:*",
      "cpe:2.3:a:gitlab:gitlab:*:*:*:*:*:*:*:*"
    ],
    "severity": "High"
  },
  "details": "An issue has been discovered in GitLab CE/EE affecting all versions starting with 12.6. Under a special condition it was possible to access data of an internal repository through a public project fork as an anonymous user.",
  "id": "BIT-gitlab-2021-22200",
  "modified": "2025-04-03T14:40:37.652Z",
  "published": "2024-03-06T11:20:07.025Z",
  "references": [
    {
      "type": "WEB",
      "url": "https://gitlab.com/gitlab-org/cves/-/blob/master/2021/CVE-2021-22200.json"
    },
    {
      "type": "WEB",
      "url": "https://gitlab.com/gitlab-org/gitlab/-/issues/247523"
    },
    {
      "type": "WEB",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2021-22200"
    }
  ],
  "schema_version": "1.7.3"
}